cherrypicks.go

```diff
@@ -140,33 +144,48 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
 	return nil
 }
 
-func cherryPickToBranch(log *logrus.Entry, comment *github.IssueCommentEvent, pr *github.PullRequest, conf *config, targetBranch string,
-	client clientgithub.Client,
-) (*github.PullRequest, error) {
+func isBottable(repoName string, conf *config, pr *github.PullRequest, targetBranch string) bool {
+	_, state, err := cherryIsPortableNoConflicts(repoName, conf, pr, targetBranch)
+	state.Cleanup()
+	if err != nil {
+		logrus.Errorf("isBottable received error: %s", err.Error())
+	}
+	return err == nil
+}
 
+func cherryIsPortableNoConflicts(repoName string, conf *config, pr *github.PullRequest, targetBranch string) (string, *git.State, error) {
```

Comment:
(related to my previous comment) I would then name this one `tryCherryPickToBranch` or something like that, because we are both using it as a dry-run (from `isBottable` (sic)) and as an actual action from `cherryPickToBranch`.

Reply:
`cherryPickToBranch` it is